### PR TITLE
geo_list class to output from geojson_list.json

### PR DESCRIPTION
**Pull request by ateucher**

### CODE REVIEW

**Comment by sckott** [comment]
:rocket:

File: R/geojson_list.R [hunk 1]
```diff
@@ -299,8 +299,9 @@ geojson_list.geo_list <- function(input, lat = NULL, lon = NULL, group = NULL,
 #' @export
 geojson_list.json <- function(input, lat = NULL, lon = NULL, group = NULL,
                               geometry = "point", type = "FeatureCollection", ...) {
-  
-  jsonlite::fromJSON(input, FALSE, ...)
+
+  output_list <- jsonlite::fromJSON(input, FALSE, ...)
+  as.geo_list(output_list, from = "list")
```
Comment (sckott) [inline R/geojson_list.R]: should be `from = "json"`

**Comment by sckott** [comment]
i'll change it when i pull down, thanks again @ateucher !

**Comment by ateucher** [comment]
Oh, duh. Thanks!

**Pull request merged**